`manylinux` wheel builds failing

### Describe the bug

Tests on the manylinux wheel builds are failing, because they can't import `cv2`. e.g. https://github.com/sunpy/sunpy/actions/runs/3806622178/jobs/6475753826. Error is:

```python
import cv2
  E           ImportError: /lib64/libz.so.1: version `ZLIB_1.2.9' not found (required by /tmp/tmp.sIYIruBBDr/venv/lib/python3.8/site-packages/cv2/../opencv_python.libs/libpng16-186fce2e.so.16.37.0)
```

### To Reproduce

import sunpy

sunpy.map.Map(...)
etc


### Screenshots

_No response_

### System Details

_No response_

### Installation method

_No response_

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -11,6 +11,7 @@ build-backend = 'setuptools.build_meta'
 # Until we have h5py binaries we can't test wheels on Python 3.11
 [tool.cibuildwheel]
 test-skip = "cp311-*"
+test-requires = ["opencv_python==4.6.0.66"]
 
 [ tool.gilesbot ]
   [ tool.gilesbot.circleci_artifacts ]

